doc/source/conf.py

```diff
@@ -94,7 +94,7 @@ def run_apidoc(_):
     source = Path(__file__).parent
     docs_dest = source / "api-reference"
     package = source.parents[1] / "src" / "qibosoq"
-    apidoc.main(["--module-first", "-o", str(docs_dest), str(package)])
+    apidoc.main(["--module-first", "-f", "-o", str(docs_dest), str(package)])
```

Comment:
Otherwise I get things like: 
```
File doc/source/api-reference/qibosoq.rst already exists, skipping.
```

And then it's not useful to have them automatically compiled every time

Reply:
Ok, I guess that there has been a problem in the transition: since they are now generated, I would gitignore these files.

Provided that, in a fresh clone (like it happens in the CI) they will always be generated.
If instead you're generating locally, we can add a task (e.g. to Poethepoet) that remove that folder before generating docs. When you're working on the docs, and not on the docstrings, there is no need to regenerate it every time. Otherwise, you'll just use the regenerating task.

Reply:
In any case, I'd move this in a separate issue, and solve in a (tiny) and stand-alone PR.

Reply:
Ok, for the moment I reverted back to the original (without -f)